src/components/Form/components/FormPassword/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { useForm } from 'react-hook-form'
+import { Errors, FormItem, Label, FormInput, HiddenError } from '../../styled'
+
+type Props = {
+	passwordName: string
+	passwordLabel: string
+	register: ReturnType<typeof useForm>['register']
+	errors: ReturnType<typeof useForm>['errors']
+	passwordErrorDisplay: boolean
+	onFocus?: () => void
+	validate?: (value: string) => string | boolean
+}
+
+const passwordRegex = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{6,}$/
+
+const FormPassword: React.FC<Props> = ({
+	passwordName,
+	passwordLabel,
+	register,
+	errors,
+	passwordErrorDisplay,
+	onFocus,
+	validate,
+}) => (
+	<>
+		<FormItem>
+			<Label htmlFor="password">{passwordLabel}</Label>
+			<FormInput
+				name={passwordName}
+				type="password"
+				placeholder="1SafePassword%"
+				ref={register({
+					required: true,
+					pattern: passwordRegex,
+					validate,
+				})}
+				onFocus={onFocus}
+			/>
+		</FormItem>
+
+		{passwordErrorDisplay ? (
+			<Errors>
+				{errors?.passwordConfirm?.types?.validate && (
+					<p>Hesla se neshodují!</p>
+				)}
+			</Errors>
+		) : (
+			<HiddenError>
+				{errors?.passwordConfirm?.types?.validate && (
+					<p>This message is never visible.</p>
+				)}
+			</HiddenError>
+		)}
```

Comment:
Why create a message that is not visible?
```suggestion
		{passwordErrorDisplay && 
			<Errors>
				{errors?.passwordConfirm?.types?.validate && (
					<p>Hesla se neshodují!</p>
				)}
			</Errors>
		}
```
If there is an error, an error message will appear. Nothing else. 🙂

Reply:
Good catch, I didn't realise I could do this! :heart: I'm not committing suggestion because with this refactor, I'll also erase HiddenError style and import. Fixed!